frontend/src/metabase/hoc/Title.jsx

```diff
@@ -53,7 +54,12 @@ const title = documentTitleOrGetter => ComposedComponent =>
           // the title. When that promise resolves, we call
           // `documentTitleOrGetter` again.
           this._documentTitle = result.title;
-          result.refresh.then(() => this._updateDocumentTitle());
+          result.refresh?.then(() => this._updateDocumentTitle());
+
+          //Getter can also return a priority index used for sorting the component stack
```

Comment:
nip: we usually have a whitespace between `//` and the beginning of an actual comment